clouddriver/clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/artifact/ArtifactReplacer.java

```diff
@@ -59,6 +62,15 @@ public class ArtifactReplacer {
 
   public ArtifactReplacer(Collection<Replacer> replacers) {
     this.replacers = ImmutableList.copyOf(replacers);
+    ObjectMapper mapper = Jackson2ObjectMapperBuilder.json().build();
```

Comment:
Is this the right mapper to use?  Don't we already pull in some object mappers fort his?  There's ... a few interesting flags particularly being JSON that should consider LIKE ignoring the unknown fields.  These may be set already... just wondering ;)

Reply:
That should be the appropriate one. It's just the spring one that we register which is the primary bean